### DISCUSSION TITLE
Handle credentials in Custom Plugins

### DISCUSSION THREAD

**Discussion by robodevchampion** [lf-edge/ekuiper#3066] [Q&A]
Hi, I see username and password in case of MQTT plugin is secure string like it becomes ***, I checked its json file haven't found anything specific, can you guide how can achieve this in my custom plugin.

Thanks

**Comment by ngjaying** [comment]
If your property name is password, the content should be hidden automatically.

**Reply by robodevchampion** [reply]
what if I have 2 password like property like access key and secret key for AWS ?
